docs/src/sway-program-types/predicates.md

```diff
@@ -10,3 +10,8 @@ fn main() -> bool {
     true
 }
 ```
+
+## Debugging predicates
+
+- Because they don't have any side-effects (they are _pure functions_), predicates *cannot* create receipts (which are transactions), and thus cannot have logging or create a backtrace. Which means no native way to debug aside from single-stepping debuggers.
```

Comment:
```suggestion
- Because they don't have any side effects (they are _pure_), predicates *cannot* create receipts (which are transactions), and thus cannot have logging or create a backtrace. This means that there is no native way to debug them aside from using a single-stepping debugger.
```